examples/bri-3/zeroKnowledgeKeys/scripts/ptau.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -e
+
+# --------------------------------------------------------------------------------
+# Phase 1
+# ... non-circuit-specific stuff
```

Comment:
let's be more precise here if possible, or just say that this is a phase that is not depending on or circuit specific

Reply:
Will update in the next PR.